OpenSim/Examples/ReadXsensData/CMakeLists.txt

```diff
@@ -4,3 +4,6 @@ set_target_properties(readXsensData
        PROPERTIES        
        PROJECT_LABEL "Utilities - readXsensData"
 )
+if(OPENSIM_BUILD_INDIVIDUAL_APPS)
+    install(TARGETS readXsensData DESTINATION "${CMAKE_INSTALL_BINDIR}")
+endif()
```

Comment:
I don't think it makes sense to install an example executable into the install bin directory. If this is a capability we want in OpenSim, and not just an example, it should be incorporated into the unified command-line tool.

Reply:
Same for APDM.

Reply:
Thanks for the thorough review @chrisdembia I totally agree and sorry if that was implicit that these standalone programs will go away and be absorbed in the opensense utility/command-line. This was intended for the demo purposes as entry point(s). We can drop now or after merge to the branch and while doing the refactoring to final layout of opensense, Do you have a preference?

Reply:
I don't see a separate code base for OpenSense. I would prefer making the change now, but do whatever is best for OpenSense's timeline.